rules/windows/credential_access_lolbas_dump_cmdline.toml

```diff
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2020/11/24"
+maturity = "production"
+updated_date = "2020/11/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of known Living Off The Land Binaries and Scripts (LOLBAS) often abused to dump LSASS memory or the Active Directory
+database (NTDS.dit) in preparation for credential access.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Potential Credential Access via LOLBAS"
+references = ["https://lolbas-project.github.io/"]
+risk_score = 73
+rule_id = "00140285-b827-4aee-aa09-8113f58a08f3"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+/* update here with any new lolbas with dump capability */
+(process.pe.original_file_name == "procdump" and process.args : "-ma") or
```

Comment:
So for this rule, do we want to focus on only LOLBAS that come pre-installed with Windows or any tooling that can be used to dump lsass or the dit file?  For example like procdump, that is a sysinternal tool right and would need to be dropped on disk first?

Reply:
any as long as it has some unique cmdline that can help us identify memory dumping activity and also one can play with parent process path to avoid FPs.

Reply:
That's fine then I think we should remove the LOLBAS aspect since it sort of conflicts with it

Reply:
For example, like procdump is not listed in the official [LOLBAS](https://lolbas-project.github.io/#) list

Reply:
Ah, I see its only 1 of many, if you have a better rule name please feel free to suggest one.